### PR TITLE
fix: config setup for hetzner k8s

### PR DESCRIPTION
**Pull request by gacevicljubisa**

### CODE REVIEW

**Review by istae — APPROVED** [review]

File: config/config.yaml [hunk 1]
```diff
@@ -81,15 +67,14 @@ node-groups:
       nginx.ingress.kubernetes.io/session-cookie-name: "SWARMGATEWAY"
       nginx.ingress.kubernetes.io/session-cookie-path: "default"
       nginx.ingress.kubernetes.io/ssl-redirect: "true"
-    ingress-class: "nginx-internal"
+      nginx.ingress.kubernetes.io/whitelist-source-range: "142.132.221.245/32" #VPN IP, or your IP (but ethswarm.org is needed)
```
Comment (darkobas2) [inline config/config.yaml:70]: Not needed now, vpn is whitelisted globally

Comment (gacevicljubisa) [reply config/config.yaml:70]: Still getting the `403 Forbidden`if I do not whitelist... Does it depend on VPN config?

File: config/staging.yaml [hunk 2]
```diff
@@ -31,4 +31,10 @@ node-groups:
 bee-configs:
   staging:
     _inherit: "default"
-    bootnodes: "/dnsaddr/mainnet.ethswarm.org"
+    swap-endpoint: http://rpc-sepolia-haproxy.default.svc.swarm1.local
```
Comment (gacevicljubisa) [inline config/staging.yaml:34]: @darkobas2 can we use this endpoint in case of setting up "our" staging env that will be connected to sepolia?

Comment (darkobas2) [reply config/staging.yaml:34]: This will work on our kubernetes clusters... Not on a local cluster or so.

File: config/config.yaml (hunk 1 above)
Comment (darkobas2) [reply config/config.yaml:70]: No, its intended to resolve testnet.internal... maybe you still have a 'public' ingress

File: config/staging.yaml (hunk 2 above)
Comment (gacevicljubisa) [reply config/staging.yaml:34]: This setup is for our kubernetes clusters, for example to deploy it in some of our namespaces..

Comment (darkobas2) [reply config/staging.yaml:34]: Yes

File: config/config.yaml (hunk 1 above)
Comment (gacevicljubisa) [reply config/config.yaml]: @darkobas2 whitelist ingress anotation is removed. After the changes you made, I have tested configuration and it works without it.

**Review by darkobas2 — APPROVED** [review]

**Pull request merged**